src/install.js

```diff
@@ -41,8 +49,13 @@ async function selectCompatibleVersion(versionSpec) {
     
     const versions = response.data;
     for (let i = 0; i < versions.length; i++) {
-        if (versionSpec === undefined || versionSpec === "latest"
-            || semver.satisfies(versions[i].tag_name, versionSpec)) {
+        if (versions[i].draft || versions[i].assets?.length === 0) {
```

Comment:
```suggestion
        if (versions[i].draft) {
```

I think there's always assets, no? I thought GitHub by default includes the source code zip/tar.gz files as part of a release? Maybe that check is unnecessary then.

Reply:
On the website, it looks as an asset.
In the API this is a dedicated field.

=>  https://api.github.com/repos/saucelabs/saucectl-run-action/releases

I'll suggest to keep for the edge case of "Release binaries being built latency".

Reply:
Ohh! Good to know 👍